components/collector/tests/source_collectors/quality_time/test_metrics.py

```diff
@@ -13,7 +13,7 @@ class QualityTimeMetricsTest(QualityTimeTestCase):
     def setUp(self):
         """Set up test data."""
         super().setUp()
-        self.api_url = f"{self.url}/api/v3/report"
+        self.api_url = f"{self.url}/api/internal/report"
 
```

Comment:
I think the Quality-time collector should use the public API. In theory the source Quality-time could be a different version so it's better to go via a versioned endpoint.